### ISSUE TITLE
Code Line Highlighting Does Not Work

### ISSUE THREAD

**Issue by jrpetersjr** [Lednerb/bilberry-hugo-theme#615]
I'm attempting to highlight a couple lines of code for a blog post and with this theme the lines are not highlighted and this can be shown with the examples from [Hugo ](https://gohugo.io/content-management/syntax-highlighting/#highlighting-in-code-fences). I'll add the code and screenshots below:

1) Example code with line highlighting
```` markdown
```go {linenos=table,hl_lines=[8,"15-17"],linenostart=199}
// GetTitleFunc returns a func that can be used to transform a string to
// title case.
//
// The supported styles are
//
// - "Go" (strings.Title)
// - "AP" (see https://www.apstylebook.com/)
// - "Chicago" (see https://www.chicagomanualofstyle.org/home.html)
//
// If an unknown or empty style is provided, AP style is what you get.
func GetTitleFunc(style string) func(s string) string {
  switch strings.ToLower(style) {
  case "go":
    return strings.Title
  case "chicago":
    return transform.NewTitleConverter(transform.ChicagoStyle)
  default:
    return transform.NewTitleConverter(transform.APStyle)
  }
}
```
````
and what is produced:
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/9285616/efcdea2b-0b3c-4444-82cc-b874794b6967)

2) Example code without the highlighting
```` markdown
```go
// GetTitleFunc returns a func that can be used to transform a string to
// title case.
//
// The supported styles are
//
// - "Go" (strings.Title)
// - "AP" (see https://www.apstylebook.com/)
// - "Chicago" (see https://www.chicagomanualofstyle.org/home.html)
//
// If an unknown or empty style is provided, AP style is what you get.
func GetTitleFunc(style string) func(s string) string {
  switch strings.ToLower(style) {
  case "go":
    return strings.Title
  case "chicago":
    return transform.NewTitleConverter(transform.ChicagoStyle)
  default:
    return transform.NewTitleConverter(transform.APStyle)
  }
}
```
````

and what is produced for that:
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/9285616/d94747a6-6f98-41fd-a912-a08ceed74984)

As you can see something is happening with the code that is produced, but it's not what supposed to be happening according to the documentation.

TIA

**Comment by atsuyaw** [comment]
It reproduced in my local environment.
Thanks for your report.

This issue seems to be caused by this line.
https://github.com/Lednerb/bilberry-hugo-theme/blob/aff9d89cbc3a82adaf5df40af84b8b2eece88507/v4/assets/sass/_articles.scss#L179-L182
Once removing the line, the following could be obtained.

![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/68371029/76412aac-839c-4f8a-abbc-6473becd269a)

Displaying line numbers also seems not working correctly.

**Comment by igor-baiborodine** [comment]
@jrpetersjr, fixed in [v4.2.2](https://github.com/Lednerb/bilberry-hugo-theme/releases/tag/v4.2.2) release.
